### DISCUSSION TITLE
Make longtblr footer always flush with bottom?

### DISCUSSION THREAD

**Discussion by ChrisJMacdonald** [lvjr/tabularray#44] [Q&A]
Hi there,
I'm working on a form with `longtblr` which has a product list that spans over several pages and product rows have different widths.
Is there a way or a work around I can use to make the footer always move to the very bottom of the page, like with a `vfill` of some sort? 

[Similar question with `longtable` ](https://tex.stackexchange.com/questions/142168/longtable-package-stretch-height-of-last-row-on-page/280366)

Thanks for all your hard work Lvjr, I'm loving the tblr package

**Comment by lvjr** [comment]
Similar to your previous question on tex.se: https://tex.stackexchange.com/questions/606734/tabularray-with-longtblr-over-duplex-printing
I think `\def\othernewpage{\vfill Footer\oldnewpage}` may be what you need.

**Reply by ChrisJMacdonald** [reply]
That works well if the longtblr finishes in the beginning/middle of the page, but if the rows extend to the bottom of the page the longtblr won't be able to account for the height of the footer and will instead push it onto the next page. 
Can I make longtblr jump to the next page prematurely? So it somehow leaves space for the footer?

For example, the addition of the 10th item would skip to the next page, not push the footer

![image](https://user-images.githubusercontent.com/39290898/127794125-89700d25-3b17-4932-bd36-ddcb9276eadc.png)
![image](https://user-images.githubusercontent.com/39290898/127794162-e5fb8a1a-1db9-425f-b9d3-c4d01f9689f2.png)

**Reply by lvjr** [reply]
Maybe you could try another approach: using `fancyhdr` package to put your footer to the page foot.

**Reply by ChrisJMacdonald** [reply]
I'll give it a try, but my immediate thought is that will make the foot show up on the blank filler pages I have for the duplex printing

**Reply by lvjr** [reply]
I think you could set different page foots for even and odd pages. But I am not so familiar with these.
